app/models/user.rb

```diff
@@ -2,6 +2,6 @@
 
 class User < ApplicationRecord
   include GDS::SSO::User
-
+  has_many :documents, foreign_key: :creator_id, inverse_of: :user, dependent: :restrict_with_exception
```

Comment:
I think we should be OK without the inverse_of, as it's only required when we change the user, which isn't very likely.

Reply:
Rubocop gets upset and demands the `:inverse_of` option if I leave it out 🙈

Reply:
Ah, OK. In that case, I think the documentation says it the inverse_of for user should be 'creator' rather than 'user'.

Reply:
Ah yes good spot, thanks.